### ISSUE TITLE
58410818 - Contra

### ISSUE THREAD

**Issue by Blackbird88** [xenia-project/game-compatibility#116]
[Marketplace](http://marketplace.xbox.com/en-US/Product/Contra/66acd000-77fe-1000-9115-d80258410818)

* Crashes after a while in a menu.
* Graphic artifacts are present

**Comment by Razzile** [comment]
state-menu ?

**Comment by Blackbird88** [comment]
Yeah could be. The menu is usable, but it crashes after set period of time regardless where you are.

**Comment by Blackbird88** [comment]
Game is in-game now, but with horrible GPU glitches
![image](https://cloud.githubusercontent.com/assets/4414625/11001525/da185a40-84a7-11e5-897a-25c16db2f2d9.png)

**Comment by KyoshiroTasya** [comment]
As if Sept 20th, 2016, the new build allows gameplay, however it's still using the incorrect shaders.

Information: Game appears blue throughout and no orange/red, bullets don't get drawn
![](https://i.gyazo.com/e751a06de6517eaf9e7edb0f3768865e.png)
![](https://i.gyazo.com/6308abb8d95a2411079376fad757fb17.png)
Suggested Tags: (state-gameplay) (gpu-shader-errors)

**Comment by bdeimos** [comment]
Tested on (https://github.com/xenia-project/xenia/commit/f89b4626a37dd4d1e5a817909252d3d4dd054299)

Playable without graphical issues. No sound.

(state-gameplay apu-silent)

[xenia.zip](https://github.com/xenia-project/game-compatibility/files/2150683/xenia.zip)

![untitled](https://user-images.githubusercontent.com/40533288/42111450-3867e220-7bed-11e8-9cbf-3e63900d302d.jpg)

**Comment by Sleepy93** [comment]
# Update
Just update the logs
The game is playable with a little gpu drawing corruption in the menu.

# Tested 
On (https://github.com/xenia-project/xenia/commit/c43ccc073da10fcf2ba43c4f90746cbc9b26880d)

# Issues
Writes to unk. register
- 0081
- 0082
- 1E4E

# Log file
[Kontra.log](https://github.com/xenia-project/game-compatibility/files/3980054/Kontra.log)

![kontra_menu](https://user-images.githubusercontent.com/20132031/71114365-e4469800-21cf-11ea-8291-cf5294d7ec56.png)

In the menu some verts are overlapped themselfs and flashing.

# Labels
- state-playable
- gpu-corrupt-drawing (just in the menu)

**Comment by Sleepy93** [comment]
Update
Game is working great
I think gpu-related issues should be removed

Tested
On (xenia-project/xenia@0b4f5ef28)

Log file
[xenia.log](https://github.com/xenia-project/game-compatibility/files/10170317/xenia.log)

Labels
state-playable

screenshots
![Xenia (master@0b4f5ef28 on Dec  3 2022) _  58410818 v2 0 1  Contra _Direct3D 12 - RTV_DSV_ 06_12_2022 22_01_55](https://user-images.githubusercontent.com/20132031/206022658-a8cdeff3-ef9e-4b04-b1ca-d76b9307e25d.png)
![Xenia (master@0b4f5ef28 on Dec  3 2022) _  58410818 v2 0 1  Contra _Direct3D 12 - RTV_DSV_ 06_12_2022 22_02_00](https://user-images.githubusercontent.com/20132031/206022667-c2463ba5-b1f1-47df-842d-a48d9a61f94f.png)
![Xenia (master@0b4f5ef28 on Dec  3 2022) _  58410818 v2 0 1  Contra _Direct3D 12 - RTV_DSV_ 06_12_2022 22_02_09](https://user-images.githubusercontent.com/20132031/206022673-4c2ceff2-ba04-4c43-91dd-baadd4a0da95.png)
![Xenia (master@0b4f5ef28 on Dec  3 2022) _  58410818 v2 0 1  Contra _Direct3D 12 - RTV_DSV_ 06_12_2022 22_02_14](https://user-images.githubusercontent.com/20132031/206022677-bf3a1a91-ca87-4981-8241-dd67740edb36.png)
![Xenia (master@0b4f5ef28 on Dec  3 2022) _  58410818 v2 0 1  Contra _Direct3D 12 - RTV_DSV_ 06_12_2022 22_02_26](https://user-images.githubusercontent.com/20132031/206022715-c4cc196c-58ad-425d-8668-31cf77508cfd.png)
